### PR TITLE
Admin sign up

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#274 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/modules/organisation/controllers/get-organisation-by-id.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ const getOrganisationById = async (req, res, next) => {
   const { id } = req.params;
   try {
     const organisation = await Organisation.getOrganisationById({ id });
+
+    console.log(`organisation`, organisation);
```
Comment (fadeomar) [inline server/modules/organisation/controllers/get-organisation-by-id.js:8]: remove console

File: server/helpers/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import verifyPassword from './verify-password';
 import matchMediaTypes from './match-media-types';
 import setMediaFileUrl from './set-media-file-url';
 import secondsToHms from './secondsToHms';
+import convertOrgNameIntoSlug from './convert-org-name-into-slug';
```
Comment (fadeomar) [inline server/helpers/index.js:8]: **SUGGESTION : 
can you change this to be stringToSlug or textToSlug

**Review by fadeomar — APPROVED** [review]

File: server/modules/organisation/controllers/get-organisation-by-id.js (hunk 1 above)
Comment (RamyAlshurafa) [reply server/modules/organisation/controllers/get-organisation-by-id.js:8]: done

File: server/helpers/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply server/helpers/index.js:8]: done

**Pull request merged**